packages/legacy-package-management/package.json

```
@@ -88,6 +87,7 @@
     "tar": "6.2.1",
     "temp-dir": "1.0.0",
     "tempy": "1.0.0",
+    "tinyglobby": "0.2.9",
```

I'd recommend upgrading to 0.2.10 here and in the other `package.json` files
```suggestion
    "tinyglobby": "0.2.10",
```

---

Totally, I would wait for a review/approval first though.